.travis.yml

```diff
@@ -4,6 +4,13 @@ language: node_js
 node_js:
   - 6
 
+cache:
+  directories:
+    - $HOME/.cache/pip
```

Comment:
Is this needed? I don't think `pip` is used in this project anywhere. Also, why is `node_modules` being removed from cache?

Reply:
Sorry, I have made a mistake here. This is not needed. Working on it.